src/main/java/Duke.java

```diff
@@ -1,10 +1,175 @@
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println(list[index]);
+                printLineSeparator();
+                break;
+            case "todo":
+
+                //Extracts the description, creates a Task object and stores the task in the list
+                Task todo = getTodoDetails(input);
+
+                printLineSeparator();
+                System.out.println("Got it. I've added this task:\n" + todo + "\nNow you have " + count + " tasks in the list.");
+                printLineSeparator();
+                break;
+            case "deadline":
+
+                //Extracts the description and day/date, creates a Task object and stores the task in the list
+                Task deadline = getDeadlineDetails(input);
+
+                printLineSeparator();
+                System.out.println("Got it. I've added this task:\n" + deadline + "\nNow you have " + count + " tasks in the list.");
+                printLineSeparator();
+                break;
+            case "event":
+
+                //Extracts the description and the time, creates a Task object and stores the task in the list
+                Task event = getEventDetails(input);
+
+                printLineSeparator();
+                System.out.println("Got it. I've added this task:\n" + event + "\nNow you have " + count + " tasks in the list.");
+                printLineSeparator();
+                break;
+            default:
+
+                //Shows invalid command incase no matching commands are given
+                printLineSeparator();
+                System.out.println("Invalid command: " + input);
+                System.out.println("Looks like you have to try again");
+                printLineSeparator();
+            }
+        }
+    }
+
+    private static String getUserInput(Scanner in) {
+        String input;
+        input = in.nextLine();
+
+        return input;
+    }
+
+    private static int getIndex(String input) {
+        String[] temp = input.split(" ");
+        int index = Integer.parseInt(temp[1]);
+        index = index - 1;
+
+        return index;
+    }
+
+    private static Task getTodoDetails(String input) {
+        String todoDescription = input.substring(4).trim();
+
+        Task todo = new Todo(todoDescription);
+        list[count] = todo;
+        count++;
+
+        return todo;
+    }
+
+    private static Task getDeadlineDetails(String input) {
+        int endIndex = input.indexOf("/");
+        String deadlineDescription = input.substring(8, endIndex);
+        String deadlineDate = getDateFromCommand(input);
+
+        Task deadline = new Deadline(deadlineDescription, deadlineDate);
+        list[count] = deadline;
+        count++;
+
+        return deadline;
+    }
+
+    private static Task getEventDetails(String input) {
+        int endIndex = input.indexOf("/");
+        String eventDescription = input.substring(5, endIndex);
```

Comment:
numbers used can be defined as constants, and maybe can add a comment to explain why number 5

Reply:
Thank you for the suggestion! 😃